packages/stan/src/derived.js

```diff
@@ -182,10 +156,10 @@ export const createDerivedAtom = ( resolver, updater = noop, config = {} ) => (
 			);
 		},
 		resolve,
-		subscribe( listener ) {
+		async subscribe( listener ) {
```

Comment:
Maybe this should return synchronously with an array of promise and unsubscribe callback?